src/css/styles/_top.scss

```diff
@@ -56,12 +81,19 @@
   display: inline-block !important;
 }
 
+.l-center-vertical--mobile-menu {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
```

Comment:
I reverted styling, to center mobile menu vertically, but IMO it looks better and more natural without this styling. To be discussed.